app/models/concerns/findings/unconfirmed.rb

```diff
@@ -1,39 +1,56 @@
 module Findings::Unconfirmed
   extend ActiveSupport::Concern
 
-  included do
-    scope :unconfirmed_for_notification, -> {
+  module ClassMethods
+    def unconfirmed_for_notification days
       where(
         [
-          'first_notification_date >= :stale_unconfirmed_date',
+          'first_notification_date = :stale_unconfirmed_date',
           'state = :state',
           'final = :false'
         ].join(' AND '),
         {
           state: Finding::STATUS[:unconfirmed],
           false: false,
-          stale_unconfirmed_date: FINDING_DAYS_FOR_SECOND_NOTIFICATION.business_days.ago.to_date
+          stale_unconfirmed_date: days.business_days.ago.to_date
         }
       )
-    }
-  end
+    end
 
-  module ClassMethods
     def notify_for_unconfirmed_for_notification_findings
-      # Sólo si no es sábado o domingo
       if Time.zone.today.workday?
-        Finding.transaction do
-          users = Finding.unconfirmed_for_notification.inject([]) do |u, finding|
-            u | finding.users.select do |user|
-              user.notifications.not_confirmed.any? do |n|
-                n.findings.include?(finding)
-              end
+        finding_days_for_next_notifications_parameters.each do |organization, value|
+          Current.organization = organization
+          Current.group        = organization.group
+          days_array           = value.to_s.split(',').map { |v| v.strip.to_i }
```

Comment:
```suggestion
          days_array           = value.to_s.split(',').map &:to_i
```
Funciona igual sin strip, por más que venga un `'    8  '` lo convierte a `8`.